src/include/ipc/header.h

```diff
@@ -329,7 +329,7 @@ struct ipc_cmd_hdr;
 #define SOF_IPC_MESSAGE_ID(x)			((x) & 0xffff)
 
 /** Maximum message size for mailbox Tx/Rx */
-#define SOF_IPC_MSG_MAX_SIZE			384
```

Comment:
I think this needs a separate commit and rationale for the increase. This affects all IPC3 implementations as well and increases memory use as this is directly used to allocate memory (which so far has been 384 bytes). 

Or is this something only for IPC4. Then tihs could be changed only if built for IPC4.

Reply:
@gbernatxintel We can play safe here. e.g.
```
#if IPC4
#define SIZE 0x1000
#else
#define SIZE 384
#endif
```

Reply:
IPC fuzz failed only for IPC3 so please try this @gbernatxintel

Reply:
Done

Reply:
@ujfalusi @plbossart This SOF_IPC_MSG_MAX_SIZE is only used in IPC3 code, so if this change is ifdeffed like this, I don't think we need to bump the ABI ver

Reply:
@kv2019i, the kernel uses different defines for the message size (in mailbox):
include/sound/sof/header.h:#define SOF_IPC_MSG_MAX_SIZE                 384
include/sound/sof/ipc4/header.h:#define SOF_IPC4_MSG_MAX_SIZE                   4096

sound/soc/sof/ipc3.c:   sdev->ipc->max_payload_size = SOF_IPC_MSG_MAX_SIZE;
sound/soc/sof/ipc4.c:   sdev->ipc->max_payload_size = SOF_IPC4_MSG_MAX_SIZE;